### PR TITLE
[Estuary] Skin sync

### PR DESCRIPTION
**Pull request by phil65**
@ronie

### CODE REVIEW

File: addons/skin.estuary/1080i/View_502_FanArt.xml [hunk 1]
```diff
@@ -205,33 +205,33 @@
...
-							<param name="icon" value="lists/year.png" />
-							<param name="label" value="$INFO[ListItem.Year]" />
-						</include>
-						<include name="InfoFlag">
-							<param name="visible" value="!IsEmpty(ListItem.Premiered)" />
-							<param name="icon" value="lists/year.png" />
-							<param name="label" value="$INFO[ListItem.Premiered]" />
-						</include>
-					</control>
+				<control type="grouplist">
```
Comment (ronie) [inline addons/skin.estuary/1080i/View_502_FanArt.xml]: indenting

File: addons/skin.estuary/1080i/View_502_FanArt.xml [hunk 2]
```diff
@@ -205,33 +205,33 @@
...
+					<top>660</top>
+					<left>40</left>
+					<orientation>horizontal</orientation>
+					<width>380</width>
+					<align>center</align>
+					<include name="InfoFlag">
+						<param name="visible" value="!IsEmpty(ListItem.duration)" />
+						<param name="icon" value="frame/clock.png" />
+						<param name="label" value="$INFO[ListItem.Duration,, min]" />
```
Comment (ronie) [inline addons/skin.estuary/1080i/View_502_FanArt.xml]: not sure if it's worth to localize 'min'...

**Comment by ronie** [comment]
looks good :-)

File: addons/skin.estuary/1080i/View_502_FanArt.xml (hunk 1 above)
Comment (phil65) [inline addons/skin.estuary/1080i/View_502_FanArt.xml]: thx, will get fixed during next syncs. (sometimes I miss to re-indent after c&ping around)
Once we are in beta stage I will fix those cosmetics instantly, ok?

File: addons/skin.estuary/1080i/View_502_FanArt.xml (hunk 2 above)
Comment (phil65) [inline addons/skin.estuary/1080i/View_502_FanArt.xml]: Same for me
 I think I also use that at some other place in case we decide to localize.

Comment (ronie) [inline addons/skin.estuary/1080i/View_502_FanArt.xml]: https://github.com/xbmc/xbmc/blob/master/addons/resource.language.en_gb/resources/strings.po#L7045

guess we should do the same :-)

Comment (phil65) [inline addons/skin.estuary/1080i/View_502_FanArt.xml]: ok, will do.

**Pull request merged**